Libraries/Text/RCTTextAttributes.m

```diff
@@ -237,6 +237,23 @@ - (UIColor *)effectiveBackgroundColor
   return effectiveBackgroundColor ?: [UIColor clearColor];
 }
 
+- (NSString *)capitalize:(NSString *)text
+{
+  NSArray *words = [text componentsSeparatedByString:@" "];
+  NSMutableArray *newWords = [[NSMutableArray alloc]init];
+  NSNumberFormatter *num = [[NSNumberFormatter alloc]init];
```

Comment:
we prefer to use `new` if there's no initializer params

Reply:
Oh I didn't know this syntax. Done!